packages/utils/telemetry-utils/src/config.ts

```diff
@@ -166,6 +166,24 @@ const safeSessionStorage = (): Storage | undefined => {
 	}
 };
 
+/**
+ * Creates a wrapper on top of an existing config provider which allows for
+ * specifying feature gates if not present in the original provider.
+ *
+ * @param defaults - default feature gate configs to be used if not specified by the original provider
+ * @param original - the original config provider
+ * @returns The value for the requested feature gate from the original provider and if not present,
+ * from the specified feature gates
+ *
+ * @alpha
```

Comment:
@ChumpChief can you take another peek at the PR, Matt? Especially this part..

Reply:
LGTM in latest - thank you for making the changes!